README.md

```diff
@@ -5,368 +5,17 @@ https://www.simplereport.gov/
-yarn to build packages correctly.
-
-### Linters
-
-This project uses [eslint](https://eslint.org/), [prettier](https://prettier.io/), and [stylelint](https://stylelint.io/) as frontend linters,
-and [spotless](https://github.com/diffplug/spotless) and [google-java-format](https://github.com/google/google-java-format) for the backend.
-GitHub Actions is configured to run these linters on every pull request, so you must resolve all mismatches/errors prior to merging.
-There are a few ways to manage this:
-
-1. Run `yarn lint:write` in the `frontend/` dir, and `./gradlew spotlessApply` in the `backend/` dir, before every commit
-1. Enable the optional pre-commit hook by running `yarn lefthook install` in the root dir (install lefthook if you haven't already done so during setup by running `yarn install`)
-1. Add extensions to your code editor that runs the linters for you on save, e.g. [prettier-vscode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode), [vscode-eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), [vscode-google-java-format](https://marketplace.visualstudio.com/items?itemName=ilkka.google-java-format)
-
-You can also run the pre-commit hooks in docker using `yarn lefthook run in-docker`.
-
-### Storybook and Chromatic
-
-[Storybook](https://storybook.js.org/) is an open source tool for developing UI components in isolation for React. It makes building UIs organized and efficient.
-
-To view the Storybook locally:
-
-- Run `yarn storybook` in the `frontend/` dir
-- Visit http://localhost:6006
-
-[Chromatic](https://www.chromatic.com/) is a web-based tool for Storybook that helps speed UI
-component development. It provides regression testing and review. It also allows for publication
-of the Storybook.
-
-Changes to the Storybook are sent to Chromatic when changes to the frontend source are push to a
-any branch. The changes are automatically accepted on merge to `main`.
-
-View the [SimpleReport Storybook](https://main--60a556a7c807cc0039ec6786.chromatic.com/)
-
-## PR Conventions
-
-The convention for branch naming on this project is {firstName}/{ticketNumber}-{short-description}. This makes it easier for other developers to find your changes.
-
-If you're merging large or complex changes, it is strongly recommended that you smoke test them in `dev`, `test`, or `pentest`. These three environments are roughly the same, with small configuration changes between each (`test` sends text results for pxp while `dev` and `pentest` do not, for example.)
-
-For all changes, please ensure the PR checklist is completed before sending out for review. If you're making UI changes, make sure to screenshot and get approval from a designer or product manager, in addition to engineers.
-
-We require two reviewers per changeset, and you cannot merge until all commits have been reviewed.
-
-## Cloud Environments
```

Comment:
⚠️ Moved to https://github.com/CDCgov/prime-simplereport/wiki/Cloud-Environments as a new section